package.json

```
@@ -1,9 +1,9 @@
 {
-    "name": "delve",
-    "icon": "delve.png",
-    "displayName": "Delve",
-    "description": "Delve Syntax Highlighting for VSCode",
-    "version": "1.0.6",
+    "name": "rel",
+    "icon": "relationalai.png",
+    "displayName": "Rel",
+    "description": "Rel Syntax Highlighting for VSCode",
+    "version": "1.0.7",
```

I think that this seems like a minor change to me, so i'm going to change your version number :)
```suggestion
    "version": "1.1.0",
```